IDEOorg/steps-bot
# #162638234 As a coach, I should be notified when a client completes their work plan.

### What does this PR do?
 This PR add enables the coach to receive email notifications whenever a client completes their work plan.

### Description of Task to be completed?
- Notifies Coach whenever a client completes their Work Plan.
- Creates an Email template that is sent to the coach.

### How should this be manually tested?
On your terminal run;
- Check out into this branch `ft-notify-coach-on-workplan-completion-162638234`.
- run npm install.
- npm start.
- Login into the bot GUI and complete any task by typing  `B` to complete your work plan.

A coach will receive a descriptive email.

#### Email Template Picture .
![screenshot 2019-01-30 at 12 51 22](https://user-images.githubusercontent.com/39088834/51975709-f6262c00-2493-11e9-9b32-76d10a63a6fd.png)






What are the relevant pivotal tracker stories?
[#162638234](https://www.pivotaltracker.com/story/show/162638234)


Questions:
N/A
